src/grammar.coffee

```diff
@@ -331,8 +329,9 @@ grammar =
     o 'Literal',                                -> new Value $1
     o 'Parenthetical',                          -> new Value $1
     o 'Range',                                  -> new Value $1
+    o 'Invocation',                             -> new Value $1
     o 'This'
-    o 'Super'
+    o 'Super',                                  -> new Value $1
```

Comment:
this is the same grammar change from #4592 (to allow eg `super.x.y`), all the rest of the changes here in `grammar.coffee` are related to making `Invocation` a grammar `Value`